Controller/AdminResettingController.php

```diff
@@ -27,7 +27,14 @@ class AdminResettingController extends ResettingController
      */
     public function requestAction()
     {
-        if ($this->container->get('security.context')->isGranted('IS_AUTHENTICATED_FULLY')) {
+        if ($this->container->has('security.authorization_checker')) {
+            $securityService = $this->container->get('security.authorization_checker');
```

Comment:
Please simplify this and rename varible to `$authorizationChecker`:

```php
$authorizationChecker = $this->container->has('security.authorization_checker') ? $this->container->get('security.authorization_checker') : $this->container->get('security.context');
```

To be done on other actions.